src/background.js

```diff
@@ -12,3 +12,21 @@ chrome.runtime.onInstalled.addListener(function () {
     ]);
   });
 });
+
+// keep a list of connected content scripts so that we can tell them to toggle mute
+const ports = new Set();
+// listen to global keyboard handler events
+chrome.commands.onCommand.addListener(function (command) {
+  if (command === "toggle-meeting-mute" && ports.size > 0) {
+    for (const port of ports) {
+      port.postMessage({ toggle: "mute" });
+    }
+  }
+});
+
+// add a handler for when a Google Meet content script loads
+chrome.runtime.onConnect.addListener(function (port) {
+  console.assert(port.name == "meet");
```

Comment:
nit: add a message to this assertion, something like `"Unexpected content script connection %s", port.name)`